InvenTree/company/api.py

```diff
@@ -163,6 +164,32 @@ class ManufacturerPartDetail(generics.RetrieveUpdateDestroyAPIView):
     serializer_class = ManufacturerPartSerializer
 
 
+class ManufacturerPartAttachmentList(generics.ListCreateAPIView, AttachmentMixin):
```

Comment:
@sur5r are you sure about this order? Mixins should be placed before the base view normally - could be wrong here too.

Reply:
No I'm not. I took `PartAttachmentList` and `PartAttachmentDetail` as an example.

I just swapped the order and noticed the uploading user was not saved with the old order but is with the new order.
So I guess you are right.

But that would hint at the order in `PartAttachment*` being wrong as well. And I just check and found the user field to be empty after uploading a (normal) part attachment.

Reply:
@sur5r that is .... interesting. Do not know how something like this can happen - Mixins are one of the core concepts of the whole framework. Seems like we slept a bit on validation in the past.
Thank you for checking - I  suggest you change the order just in your new code and we fix the others in a separate PR that gets merged ASAP.